husky_description/urdf/description.xacro

```diff
@@ -31,7 +31,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:arg name="kinect_enabled" default="false" />
   <xacro:arg name="robotiq_grippers_enabled" default="$(optenv ROBOTIQ_GRIPPERS_ENABLED false)" />
  
-  <xacro:include filename="$(find husky_description)/urdf/husky.urdf.xacro" />
+  <xacro:include filename="$(find husky_description)/urdf/dual_arm_husky.urdf.xacro" />
```

Comment:
I agree - but as long as it's not being merged up into the general master branch and the husky fixed up (i.e. the elements from dual_arm_husky merged in), this is the only way to make Gazebo happy